### DISCUSSION TITLE
关于

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#7] [Comments]
# 关于

关于 ouuan 的博客

https://ouuan.moe/about

**Comment by HeliumOI** [comment]
Orz

**Comment by Chiyuru** [comment]
好的𦂕 (óu) 雨 (yǔ) 玵 (án) 老师（错乱

**Reply by ouuan** [reply]
<img width="20px" src="https://user-images.githubusercontent.com/30581822/201959231-c9cd9fe9-844c-40b3-99c2-a4627ef0c893.png">

**Comment by zyf722** [comment]
> *Good morning, and in case I don't see you, good afternoon, good evening and good night.*

**Comment by aqingaaaa** [comment]
从头到尾读完，可是找不到你的联系方式，我想联系你

**Reply by ouuan** [reply]
给了 GitHub 帐号就可以找到联系方式啊

**Comment by aqingaaaa** [comment]
嘻嘻，我找你你微信了，我还加了




祝好
青柠
17304101127






发件人：Yufan You ***@***.***>
发送日期：2023-01-04 15:51:28
收件人：ouuan/iles-blog ***@***.***>
抄送人：qingningxiaoxiaoxiao ***@***.***>,Comment ***@***.***>
主题：Re: [ouuan/iles-blog] 关于 (Discussion #7)

给了 GitHub 帐号就可以找到联系方式啊

**Comment by nick-cjyx9** [comment]
感谢大佬安利三狮，三狮真的真的是一部好作品！！！

**Comment by zhaochenyang20** [comment]
A pretty interesting thing is that I also have an high school classmate with 欧元 as his name and also an OIer.

**Comment by nick-cjyx9** [comment]
Happy New Year's Day！

**Comment by SilverRainZ** [comment]
Tuna 巨佬 Q_Q，是新时代的年轻人！

> 有时我会因为找不到项目的打赏链接或者没有支付方式而有些难过，也正因如此，我自己的项目会放打赏链接

给了我一些启发…这就给自己的项目统统挂上链接。

**Comment by raindrop2508** [comment]
三狮是真的好

**Comment by RunMintOn** [comment]
你好.
